types/__tests__/type-tests.ts

```diff
@@ -228,6 +229,18 @@ export async function testWaitFors() {
   await waitFor(async () => {})
 }
 
+export async function testWithin() {
+  const sut = within(document.body)
```

Comment:
What is "sut"?

Reply:
It's an abbreviation for "System Under Test", I couldn't think of a better name.
I can change it if you want though.

Reply:
please 😄

Reply:
No problem, renamed to `container`.